File: src/main/java/de/spries/fleetcommander/rest/ShipFormationParams.java
package de.spries.fleetcommander.rest;

public class ShipFormationParams {
	private int shipCount;
	private int originPlanetId;
	private int destinationPlanetId;

	public int getShipCount() {
		return shipCount;
	}

	public void setShipCount(int shipCount) {
		this.shipCount = shipCount;
	}

	public int getOriginPlanetId() {
		return originPlanetId;
	}

	public void setOriginPlanetId(int originPlanetId) {
		this.originPlanetId = originPlanetId;
	}

	public int getDestinationPlanetId() {
		return destinationPlanetId;
	}

	public void setDestinationPlanetId(int destinationPlanetId) {
		this.destinationPlanetId = destinationPlanetId;
	}
}

File: src/main/kotlin/de/spries/fleetcommander/web/dto/ShipFormationParams.kt
package de.spries.fleetcommander.web.dto

data class ShipFormationParams(val shipCount: Int = 0, val originPlanetId: Int = 0, val destinationPlanetId: Int = 0) {

    override fun toString(): String {
        return "ShipFormationParams($shipCount from $originPlanetId to $destinationPlanetId)"
    }
}